sdk-tests/src/test/java/io/dapr/it/DaprRun.java

```diff
@@ -58,7 +58,7 @@ private DaprRun(String testName,
                   DaprApiProtocol protocol,
                   DaprApiProtocol appProtocol) {
     // The app name needs to be deterministic since we depend on it to kill previous runs.
-    this.appName = serviceClass == null ? testName : String.format("%s_%s", testName, serviceClass.getSimpleName());
+    this.appName = serviceClass == null ? testName : String.format("%s-%s", testName, serviceClass.getSimpleName());
```

Comment:
@shubham1172 Is this change related to the change done in runtime ? That change only talk about `.` or empty string if I remember correctly.

Reply:
Ya, I think it does prohibit "_" also.

Reply:
It does talk about `.` but it does prohibit `_` and upper case letters too.